.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--
+Thank you for contributing to this project! You must fill out the information below before we can review this pull request.
+-->
+
+### Why:
+
+This PR fixes: [issue link]
+
+### What's being changed:
+
+<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->
+
+### Type of change:
+
+<!--Please delete options that are not relevant.-->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Feature or enhancements (non-breaking change which adds functionality)
+- [ ] Documentation updates (non-breaking change which updates documents)
+
+### How Has This Been Tested?
+
+<!--Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration-->
```

Comment:
You can add a link or maybe just the npm commands to prompt users how should they test it locally.